scripts/jenkins/cloud/manual/configure-deployer-container.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+- name: Create deployer container
+  hosts: ci-deployer
+  connection: docker
+  remote_user: root
```

Comment:
Does this have to be root?  or maybe require user to be added to docker group?

Reply:
Out of the box you may need root privileges to run docker operations (ie docker pull, docker run, etc.). However, that can be adjusted so root privileges aren't required.

Reply:
It was root because the default user of the SP4 image is root. I have removed that playbook now and moved this functionality into the entrypoint script.